flows/vision-zero/cris_import/cris_import.py

```diff
@@ -546,59 +611,186 @@ def align_records(typed_token, dry_run):
             else:
                 # An insert is always just an vanilla insert, as there is not a pair of records to compare.
                 # Produce the SQL which creates a new VZDB record from a query of the imported data
-                insert_statement = util.form_insert_statement(output_map, table, input_column_names, import_key_sql, DB_IMPORT_SCHEMA)
+                insert_statement = util.form_insert_statement(output_map, table, input_column_names, import_key_sql, map_state["import_schema"])
                 logger.info(f"Executing insert in {output_map[table]} for where " + record_key_sql)
 
                 # Execute the insert statement
                 util.try_statement(pg, output_map, table, record_key_sql, insert_statement, dry_run)
 
     # fmt: on
-    return True
+    return map_state
+
+
+@task(
+    name="Group CSVs into logical groups",
+)
+def group_csvs_into_logical_groups(extracted_archives, dry_run):
```

Comment:
The root of the solution for the split-schema problem; break the CSVs into sets based on their filename prefix.